src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.Mono.cs

```diff
@@ -217,12 +218,12 @@ public sealed class AssemblyBuilder : Assembly
         private bool manifest_module_used;
 
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        [DynamicDependency("RuntimeResolve", typeof(ModuleBuilder))]
```

Comment:
I've moved this attribute closer to the internalcall that depends on it. Does that make sense?